hello-kubernetes/README.md

```diff
@@ -100,10 +100,10 @@ You can also export it to a variable:
 
 ```
 Linux/MacOS
-export NODE_APP=$(kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}')
+export NODE_APP=$(kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].hostname}')
```

Comment:
The hostname field is not set for a vanilla AKS cluster:

```bash
hello-kubernetes % kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].hostname}'  
hello-kubernetes % kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}'      
51.143.63.77
```

What is the problem you are trying to solve? Does querying for the IP not work in your setup? I'd be interested to learn more about what you are running against.

Reply:
Hi @wcs1only I'm running kubernetes under Docker Desktop on Windows. In that setup the .ip property isn't set but the .hostname is set (the full details are in the Issue referenced above https://github.com/dapr/quickstarts/issues/367).

Reply:
So the opposite is true (.ip is set, but .hostname isn't) for AKS clusters. Does the optional kubectl port-forward step below, work correctly for your setup?:
https://github.com/dapr/quickstarts/pull/368/files#diff-adecd0576f327b839c17dc709d631777d8abb16cb271f81d42bc1e01885a6c11R121

If so, I think the best solution might be to just remove .ip and .hostname references entirely and just recommend that all users go the port forwarding route.

Reply:
@wcs1only Good suggestion. Yes, the hostname approach looks like it will work. Do you want to merge in your change or need me to do it?

Reply:
I'll get it. I want to test a few other k8s configurations just to make sure this approach works for all of them.